packages/contracts/contracts/voting/multisig/Multisig.sol

```diff
@@ -38,20 +38,12 @@ contract Multisig is PluginUUPSUpgradeable, ProposalUpgradeable, Addresslist {
     /// @param startDate The timestamp when the proposal starts.
     /// @param endDate The timestamp when the proposal expires.
     struct ProposalParameters {
-        uint256 minApprovals;
+        uint16 minApprovals;
         uint64 snapshotBlock;
         uint64 startDate;
         uint64 endDate;
     }
 
-    /// @notice A container for the proposal tally.
-    /// @param approvals The number of approvals casted.
-    /// @param addresslistLength The length of the addresslist.
-    struct Tally {
-        uint256 approvals;
-        uint256 addresslistLength;
```

Comment:
Where are we storing the address list length now? It could have changed after the proposal was created

Reply:
AddressListLenght wasn't used at all anywhere. :D So @Michael-A-Heuer also agreed and we removed it.